templates/server/base.html

```diff
@@ -77,6 +77,7 @@ <h5>Appreciate</h5>
 <script src="/static/js/vendor/bootstrap.min.js"></script>
 <!-- Place this tag in your head or just before your close body tag. -->
 <script async defer src="/static/js/vendor/github_buttons.js"></script>
+<script defer type="text/javascript" src="static/js/helpers.js"></script>
```

Comment:
Remove defer and load it synchronously otherwise it does not load. I just verified that locally.